dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -86,3 +89,9 @@ includePathInWASReqURL.desc=Setting the Path parameter can allow the client/brow
 
 trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
+
+overrideHttpAuthenticationMechanism=Name of the container provided HttpAuthenticationMechanism to override the application configured HttpAuthenticationMechanism. The HttpAutheMechanism is defined by the Java EE 8 Security specification.
+overrideHttpAuthenticationMechanism.desc=Specifies HttpAuthenticationMechanism name for overriding an application defined HttpAuthenticationMechanism. The acceptable value is BASIC or FORM. When FORM is used, loginFormURL and loginErrorURL need to be specified.
+
+httpBasicAuthMechanismRealmName=Realm name fot the basic authentication to be used when overrideHttpAuthenticationMechanism is enabled and set to to BASIC.
+httpBasicAuthMechanismRealmName.desc=Specifies a realm name for JavaEE security basic authentication. This is only effective when overrideHttpAuthenticationMechanism is set as BASIC. The default value is defaultRealm.
```

Comment:
Specifies a realm name for Java EE security basic authentication and is effective only when the overrideHttpAuthenticationMechanism mechanism is set as BASIC. The default value is defaultRealm.